initial-promises/node-server/common/1.0/site.cf

```diff
@@ -31,7 +31,6 @@ bundle common g
       "rudder_var"                 string => "/data/rudder";
       "rudder_curl"                string => "/system/bin/curl";
       "rudder_rm"                  string => "/system/xbin/rm";
-      "rudder_tools_files_android" slist => { "cpuid-android-V1.0.sh" };
```

Comment:
These lists of files are used in ```update_action``` bundle, which is only run in failsafe mode, so they must be in ```failsafe.cf```'s copy of the ```g``` bundle.